templates/search/results-contextual-en.html

```diff
@@ -17,7 +17,7 @@
 <div class="well">
 	<form class="mrgn-tp-sm">
 		<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
-		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search">
+		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query="safe" />
```

Comment:
@igkislev can you remove the value "safe"? That parameter value have no effect.
```suggestion
		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query />
```